product_docs/docs/epas/16/upgrading/05_performing_a_minor_version_update_of_an_rpm_installation.mdx

```diff
@@ -5,28 +5,32 @@ redirects:
 - /epas/latest/epas_upgrade_guide/05_performing_a_minor_version_update_of_an_rpm_installation/
 ---
 
-If you used an RPM package to install EDB Postgres Advanced Server or its supporting components, you can use `yum` to perform a minor version upgrade to a more recent version. To review a list of the package updates that are available for your system, open a command line, assume root privileges, and enter the command:
+If you used an RPM package to install EDB Postgres Advanced Server or its supporting components, you can use `dnf` to perform a minor version upgrade to a more recent version. 
 
-```shell
-yum check-update <package_name>
-```
+1.  To list the package updates that are available for your system, open a command line, assume root privileges, and run:
 
-Where `package_name` is the search term for which you want to search for updates. You can include wildcard values in the search term. To use `yum update` to install an updated package, use the command:
+    ```shell
+    dnf check-update <package_name>
+    ```
 
-```shell
-yum update <package_name>
-```
+    Where `package_name` is the search term for which you want to list available updates. You can include wildcard values in the search. 
+    
+1.  After you identify the package(s), update them:
 
-Where `package_name` is the name of the package you want to update. Include wildcard values in the update command to update multiple related packages with a single command. For example, use the following command to update all packages whose names include the expression `edb`:
+    ```shell
+    dnf update <package_name>
+    ```
 
-```shell
-yum update edb*
-```
+    Where `package_name` is the name of the package you want to update. Include wildcard values in the update command to update multiple related packages with a single command. For example, use the following command to update all packages with names that include the expression `edb`:
```

Comment:
nit-pick and existed prior to this edit, but... This is slightly misleading. Should be,

```suggestion
    Where `package_name` is the name of the package you want to update. Include wildcard values in the update command to update multiple related packages with a single command. For example, use the following command to update all packages with names start with `edb`:
```

For symmetry, might also include same example for step 1 above (ideally you'd know which packages an expression will update before updating them).